components/org.wso2.analytics.is/src/main/java/org/wso2/analytics/is/siddhi/extension/IpToCityExtension.java

```diff
@@ -44,7 +44,7 @@ protected Object execute(Object[] objects) {
     @Override
     protected Object execute(Object o) {
         try {
-            return GeoResolverInitializer.getInstance().getLocationResolver().getLocation((o.toString()));
+            return GeoResolverInitializer.getInstance().getLocationResolver().getLocation((o.toString())).getCity();
```

Comment:
Can we guarantee that `GeoResolverInitializer.getInstance().getLocationResolver().getLocation((o.toString()))` will not be null in any case? If not shall we check null before calling `getCity()`

Reply:
I added the null check

Reply:
Thanks!